api/github_api/util/SceGithubApiHandler.js

```diff
@@ -0,0 +1,142 @@
+const fetch = require('node-fetch');
+
+/**
+ * Handles website's backend request to interface with Github's various APIs
+ */
+class SceGithubApiHandler {
+  /**
+   * Create SceGithubApiHandler object
+   * @param {String} clientId Client ID generated from Github App
+   * @param {String} clientSecret Client secret generated from Github App
+   */
+  constructor(clientId, clientSecret) {
+    this.clientId = clientId;
+    this.clientSecret = clientSecret;
+  }
+
+  /**
+   * Converts Github date format to MM-DD-YYYY format
+   * @param {String} date Github's date string
+   * e.g. "2011-04-14T16:00:49Z"
+   * @returns {String} String of reformatted date
+   */
+  parseDate(date) {
+    const formattedDate = date.split('T')[0].split('-');
+    return `${formattedDate[1]}-${formattedDate[2]}-${formattedDate[0]}`;
+  }
+
+  /**
+   * Gets top 5 ranked users that have contributed to the given repo
+   * in the past 30 days
+   * @param {String} repoName Name of SCE Github repo
+   * @returns {Array<Object>|boolean} Array containing objects with
+   * 2 properties user and commit. Array is ordered by commit count
+   * e.g. [ {user: "user1", commits: 7}, {user: "user2", commits: 4}, ... ]
+   */
+  getContributorsInPastMonthFromRepo(repoName) {
+    return fetch(
```

Comment:
i would use `axios.get` instead, as we already have it installed. you can see it used in 
[`src/APIFunctions/Event.js:23`](https://github.com/SCE-Development/Core-v4/blob/f82138a1e5bf97c3110472782abf953e7c202bd0/src/APIFunctions/Event.js#L23)